src/autoinstrumentation.ts

```diff
@@ -38,11 +37,15 @@ import {
   SpanProcessor,
 } from "@opentelemetry/sdk-trace-node";
 import * as semver from "semver";
+import { OdigosProcessDetector } from "./OdigosProcessDetector";
 
 // not yet published in '@opentelemetry/semantic-conventions'
 const SEMRESATTRS_TELEMETRY_DISTRO_NAME = "telemetry.distro.name";
 const SEMRESATTRS_TELEMETRY_DISTRO_VERSION = "telemetry.distro.version";
 
+// This attribute is necessary for our use case to track process-specific information that is not covered by OpenTelemetry’s built-in attributes.
+const PROCESS_VPID = "process.vpid"
```

Comment:
I think it makes more sense to have it in `src/OdigosProcessDetector.ts` file where we define process resource attributes related definitions.